Games/types/Mafia/const/Priority.js

```diff
@@ -172,6 +173,7 @@ module.exports = {
     PRIORITY_GIVE_VISITORS_ITEM,
 
     // MEETING PRIORITY
+    PRIORITY_BANQUET_MEETING,
```

Comment:
the day priorities are currently useless because they don't affect each other

so i propose combining all the day priorities jail_meeting party_meeting and banquet_meeting 
to PRIORITY_INVITE_TO_NIGHT_MEETING_DEFAULT = 0